toolkit/resources/manifests/package/update_manifests.sh

```diff
@@ -259,6 +259,7 @@ generate_pkggen_core () {
         grep "^gtk-doc-" $TmpPkgGen
         grep "^autoconf-" $TmpPkgGen
         grep "^automake-" $TmpPkgGen
+        grep "^ocaml-srpms-macros-" $TmpPkgGen
```

Comment:
You must also modify `toolkit/scripts/toolchain/build_official_toolchain_rpms.sh` by adding the `build_rpm_in_chroot_no_install ocaml-srpm-macros` line. The package has no dependencies, so it can go right anywhere between `echo Building final list of toolchain RPMs` and the last build instructions.

Reply:
Added